setup.py

```diff
@@ -24,7 +24,10 @@
     ],
     include_package_data=True,
     python_requires='>=3.6',
-    install_requires=['PyYAML~=5.1'],
+    install_requires=[
+        'PyYAML~=5.1',
+        'dataclasses;python_version=="3.6"'
```

Comment:
```suggestion
        'dataclasses;python_version=="3.6"',
```

Reply:
What's the comma here for?

Reply:
Less noisy diffs in version control when a new dependency is added to the list. Without the trailing comma you would have to add a comma, which results in a +2 -1 diff rather than a +1 diff.

Reply:
Ah, I never even realized a trailing comma was valid syntax. Good to know